pyHS100/discover.py

```diff
@@ -43,6 +43,7 @@ class Discover:
     @staticmethod
     def discover(
         protocol: TPLinkSmartHomeProtocol = None,
+        target: str = "255.255.255.255",
```

Comment:
I'm not sure if that's the proper position for this in the argument list, but considering the current master has evolved and is breaking the backwards compatibility, I'd propose that we simply adjust discover to allow keyword-only arguments.

Reply:
I think this can be done in a separate PR, so let's get this merged if someone is eagerly waiting for this functionality.  #158 is at least partially related (there are mentions about multiple subnets in the comments).